assessments/projects/volcano/README.md

```diff
@@ -0,0 +1,7 @@
+# Volcano feedback
+
+## Summary
+
+## Feedback to Volcano Team
+
+## Feedback to CNCF
```

Comment:
This file is part of the joint-assessment. 
Please delete this from the self-assessment